### PR TITLE
Contact section frontend

### PR DESCRIPTION
**Pull request by Kushan1994**

### CODE REVIEW

File: resources/js/Components/Frontend/Contact/ContactForm.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                </div>
+
+                <Textarea id="message" name="message" rows="4" class="w-full mt-1" />
+            </FormGroup>
+
+            <fieldset class="sm:col-span-2 ml-4">
+                <legend class="block text-sm font-medium text-gray-700">Expected budget</legend>
+                <div class="mt-4 grid grid-cols-1 gap-y-4">
+                    <div class="flex items-center">
+                        <input id="budget-under-25k" name="budget" value="under_25k" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300" />
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactForm.vue:52]: `<Input>` do capital

File: resources/js/Components/Frontend/Contact/ContactForm.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                        <label for="budget-under-25k" class="ml-3">
+                            <span class="block text-sm text-gray-700">Less than Rs: 25 000</span>
+                        </label>
+                    </div>
+                    <div class="flex items-center">
+                        <input id="budget-25k-50k" name="budget" value="25k-50k" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300" />
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactForm.vue:58]: Capital

File: resources/js/Components/Frontend/Contact/ContactForm.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                        <label for="budget-25k-50k" class="ml-3">
+                            <span class="block text-sm text-gray-700">Rs: 25 000 – Rs: 50 000</span>
+                        </label>
+                    </div>
+                    <div class="flex items-center">
+                        <input id="budget-50k-100k" name="budget" value="50k-100k" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300" />
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactForm.vue:64]: Capital

File: resources/js/Components/Frontend/Contact/ContactForm.vue [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                        <label for="budget-50k-100k" class="ml-3">
+                            <span class="block text-sm text-gray-700">Rs: 50 000 – Rs: 100 000</span>
+                        </label>
+                    </div>
+                    <div class="flex items-center">
+                        <input id="budget-over-100k" name="budget" value="over_100k" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300" />
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactForm.vue:70]: capital

File: resources/js/Components/Frontend/Contact/ContactForm.vue [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                </div>
+            </fieldset>
+
+            <FormGroup class="sm:col-span-2">
+                <Label for="how-did-you-hear-about-us">How did you hear about us?</Label>
+                <Input type="text" name="how-did-you-hear-about-us" id="how-did-you-hear-about-us" class="w-full mt-1" />
+            </FormGroup>
+
+            <div class="sm:col-span-2 sm:flex sm:justify-end mr-4">
+                <Button type="submit">Submit</Button>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactForm.vue:84]: remove `type` attribute. submit is the default one. Do not need to mention it here

File: resources/js/Components/Frontend/Contact/Contact.vue [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<script setup>
+import ContactInformation from "@/Components/Frontend/Contact/ContactInformation";
+import ContactForm from "@/Components/Frontend/Contact/ContactForm";
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/Contact.vue:3]: rename component as Form.vue

File: resources/js/Components/Frontend/Contact/ContactInfoBg.vue [hunk 7]
```diff
@@ -0,0 +1,45 @@
+<template>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactInfoBg.vue:1]: didn't understand the filename

File: resources/js/Components/Frontend/Contact/ContactInformation.vue [hunk 8]
```diff
@@ -0,0 +1,40 @@
+<script setup>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactInformation.vue:1]: rename as `Information.vue`

File: resources/js/Components/Frontend/Contact/ContactInformation.vue [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+import {MailIcon, PhoneIcon, LocationMarkerIcon} from '@heroicons/vue/outline'
+import SocialMediaLink from "@/Components/LayoutComponents/SocialMediaLink";
+import ContactInfoBg from "@/Components/Frontend/Contact/ContactInfoBg";
+
+</script>
+<template>
+    <div class="relative overflow-hidden py-10 px-6 bg-brown-700 sm:px-10 xl:p-12">
+
+        <ContactInfoBg />
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Contact/ContactInformation.vue:10]: what is this?

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: resources/js/Components/Frontend/Contact/ContactForm.vue (hunk 1 above)
Comment (Kushan1994) [reply resources/js/Components/Frontend/Contact/ContactForm.vue:52]: eka iwara netho radio dann oni hadala

File: resources/js/Components/Frontend/Homepage/Contact/Background.vue [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+            <path d="M-99 461.107L608.107-246l707.103 707.107-707.103 707.103L-99 461.107z" fill="url(#linear1)"
+                  fill-opacity=".1"/>
+            <defs>
+                <linearGradient id="linear1" x1="254.553" y1="107.554" x2="961.66" y2="814.66"
+                                gradientUnits="userSpaceOnUse">
+                    <stop stop-color="#fff"/>
+                    <stop offset="1" stop-color="#fff" stop-opacity="0"/>
+                </linearGradient>
+            </defs>
+        </svg>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Background.vue:14]: make different component

File: resources/js/Components/Frontend/Homepage/Contact/Background.vue [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+            <path d="M-161 382.107L546.107-325l707.103 707.107-707.103 707.103L-161 382.107z" fill="url(#linear2)"
+                  fill-opacity=".1"/>
+            <defs>
+                <linearGradient id="linear2" x1="192.553" y1="28.553" x2="899.66" y2="735.66"
+                                gradientUnits="userSpaceOnUse">
+                    <stop stop-color="#fff"/>
+                    <stop offset="1" stop-color="#fff" stop-opacity="0"/>
+                </linearGradient>
+            </defs>
+        </svg>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Background.vue:29]: make different component.

File: resources/js/Components/Frontend/Homepage/Contact/Background.vue [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+            <path d="M-161 679.107L546.107-28l707.103 707.107-707.103 707.103L-161 679.107z" fill="url(#linear3)"
+                  fill-opacity=".1"/>
+            <defs>
+                <linearGradient id="linear3" x1="192.553" y1="325.553" x2="899.66" y2="1032.66"
+                                gradientUnits="userSpaceOnUse">
+                    <stop stop-color="#fff"/>
+                    <stop offset="1" stop-color="#fff" stop-opacity="0"/>
+                </linearGradient>
+            </defs>
+        </svg>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Background.vue:43]: make different component

File: resources/js/Components/Frontend/Homepage/Contact/Form.vue [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+        {id: 'budget-over-25k', value: 'over_100k', label: 'Rs: 100 000 +'},
+
+    ]
+
+
+</script>
+<template>
+    <div class="py-10 px-6 sm:px-10 lg:col-span-2 xl:p-12">
+        <h3 class="text-lg font-medium text-gray-900">Send us a message</h3>
+        <form action="#" method="POST" class="mt-6 grid grid-cols-1 gap-y-3 sm:grid-cols-2 sm:gap-x-2">
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Form.vue:22]: there is no need of `<form>` tag. remove this

File: resources/js/Components/Frontend/Homepage/Contact/Information.vue [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+            <dt><span class="sr-only">Facebook</span> </dt>
+            <dd class="flex text-base text-indigo-50">
+                <Instagram class="flex-shrink-0 w-6 h-6 text-indigo-200" aria-hidden="true" />
+                <span class="ml-3">Rn Construction</span>
+            </dd>
+        </dl>
+
+
+
+    </div>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Information.vue:54]: unnecessary white spaces

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: resources/js/Components/Frontend/Homepage/Contact/Background.vue [hunk 15]
```diff
@@ -0,0 +1,18 @@
+<script setup>
+import ImageLgVersion from "@/Components/SvgImages/VectorPath/SquareLg";
+import Image from "@/Components/SvgImages/VectorPath/Square";
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Background.vue:3]: SquareDefault

File: resources/js/Components/Frontend/Homepage/Contact/Background.vue [hunk 16]
```diff
@@ -0,0 +1,18 @@
+<script setup>
+import ImageLgVersion from "@/Components/SvgImages/VectorPath/SquareLg";
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Background.vue:2]: ImageLarge

File: resources/js/Components/Frontend/Homepage/Contact/Background.vue [hunk 17]
```diff
@@ -0,0 +1,18 @@
...
+import Image from "@/Components/SvgImages/VectorPath/Square";
+import ImageMobileVersion from "@/Components/SvgImages/VectorPath/SquareSm";
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Background.vue:4]: ImageMobile

File: resources/js/Components/Frontend/Homepage/Contact/Form.vue [hunk 18]
```diff
@@ -0,0 +1,80 @@
...
+import Textarea from "@/Components/Textarea";
+import Button from "@/Components/Button";
+import RadioButton from "@/Components/RadioButton";
+
+const budgetRanges =  [
+        {id: 'budget-under-25k', value: 'under_25k', label: 'Less than Rs: 25 000'},
+        {id: 'budget-25k-50k', value: '25K - 50K', label: 'Rs: 25 000 – Rs: 50 000'},
+        {id: 'budget-50k-100k', value: '50k-100k', label: 'Rs: 50 000 – Rs: 100 000'},
+        {id: 'budget-over-25k', value: 'over_100k', label: 'Rs: 100 000 +'},
+
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Form.vue:14]: blank line

File: resources/js/Components/Frontend/Homepage/Contact/Form.vue [hunk 19]
```diff
@@ -0,0 +1,80 @@
...
+    ]
+
+
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Form.vue:17]: blank line

File: resources/js/Components/Frontend/Homepage/Contact/Information.vue [hunk 20]
```diff
@@ -0,0 +1,50 @@
+<script setup>
+import {MailIcon, PhoneIcon, LocationMarkerIcon} from '@heroicons/vue/outline'
+import Background from "@/Components/Frontend/Homepage/Contact/Background";
+import Whatsapp from "@/Components/Icons/Whatsapp";
+import Facebook from "@/Components/Icons/Facebook";
+import Instagram from "@/Components/Icons/Instagram";
+
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Information.vue:7]: remove blank line

File: resources/js/Components/Frontend/Homepage/Contact/Information.vue [hunk 21]
```diff
@@ -0,0 +1,50 @@
...
+            <dt><span class="sr-only">Facebook</span></dt>
+            <dd class="flex text-base text-indigo-50">
+                <Facebook class="flex-shrink-0 w-6 h-6 text-indigo-200" aria-hidden="true"/>
+                <span class="ml-3">Rn Construction</span>
+            </dd>
+            <dt><span class="sr-only">Facebook</span></dt>
+            <dd class="flex text-base text-indigo-50">
+                <Instagram class="flex-shrink-0 w-6 h-6 text-indigo-200" aria-hidden="true"/>
+                <span class="ml-3">Rn Construction</span>
+            </dd>
```
Comment (sureshhemal) [inline resources/js/Components/Frontend/Homepage/Contact/Information.vue:47]: make dynamic

**Review by sureshhemal — CHANGES_REQUESTED** [review]

**Review by sureshhemal — APPROVED** [review]

**Pull request merged**